api/api.go

```diff
@@ -9,14 +9,18 @@ import (
 	"log"
 	"net/http"
 	"strconv"
+
+	"github.com/rs/cors"
 )
 
 func main() {
-	http.HandleFunc("/", func(w http.ResponseWriter, r *http.Request) {
+	mux := http.NewServeMux()
```

Comment:
Added the mutex here in order to circumvent a CORS issue when hosting the FE and the BE on localhost.